testsuite/tests/translprim/array_spec.compilers.reference.flat

```diff
@@ -22,61 +22,61 @@
       (function a/1015 x/1016 (array.unsafe_set[gen] a/1015 0 x/1016))
       (let
         (eta_gen_len/1017 =
-           (function prim/1108 stub (array.length[gen] prim/1108))
+           (function prim/1065 stub (array.length[gen] prim/1065))
```

Comment:
I believe that these changes should not be here, that they come from a typo in a9aa39d735eaae94d91e40ca65052db0ab0ef5c2. (cc @shindere)